### ISSUE TITLE
Kitematic docker vm error

### ISSUE THREAD

**Issue by msound** [docker/toolbox#286]
I am trying to run docker-toolbox 1.9.0b on OS X 10.11.1

Installation is successful, but when running Kitematic for first time, it results in an error:

![](http://msound.us/pics/49opa.png)

Full error message [here](http://pastebin.com/SekUFV3v)

More info:
```
--- ~ » VBoxManage --version    
5.0.8r103449
--- ~ » docker-machine --version
docker-machine version 0.5.0 (04cfa58)
```

**Comment by msound** [comment]
I tried deleting VM and retry setup several times, still same error.

**Comment by msound** [comment]
I tried deleting the `default` vm and creating it from command line:
```
docker-machine stop default
docker-machine rm default
```
And then I tried to create a `default` machine with the command:
```
docker-machine -D create -d virtualbox default
```

The entire log of the output is [here](http://pastebin.com/W9DFTGXh)

**Comment by mrjatx** [comment]
@msound looks like I'm able to reproduce this when VT-x is disabled. The VirtualBox container is unable to start at all. Once I enabled VT-x I was able to move on.

**Comment by msound** [comment]
I digged around some more. Installed docker-toolbox 1.9.0d. Upgraded virtual box from 5.0.8 to 5.0.10.

Still I was getting the same error.

Using VirtualBox I connected to the "default" machine and I could see that `eth1` was not getting assigned any IP address. From VirtualBox GUI I could see that this adapter was configured for "vboxnet5" in my case. 

So I went to Virtual Box menu -> preferences -> Network -> Host-only Networks -> vboxnet5 -> edit -> DHCP and found that the "Enable Server" option was not checked. So I checked it and entered some default values.

I then deleted the "default" vm, and restarted Kitematic. It created the VM again, and this time everything is working perfectly!
